web/src/constants/SemanticGroupNames.constants.ts

```diff
@@ -34,129 +24,121 @@ export const SemanticGroupNamesToText = {
 
 export const SemanticGroupNameNodeMap: Record<SemanticGroupNames, {primary: string[]; type: string}> = {
   [SemanticGroupNames.Http]: {
-    primary: ['http.target', 'service.name'],
+    primary: [Attributes.HTTP_TARGET, Attributes.SERVICE_NAME],
```

Comment:
No more magic strings, now everything comes from the OTEL semantic conventions package